content-repo/extra-docs/releases/20.6.0.md

```diff
@@ -0,0 +1,865 @@
+- Improved logging when running with the *debug-mode* argument.
+- Added support for the CVE indicator class.
+##### CommonServerPowerShell
+Removed override of the *Write-Output* function.
+
+---
+
+### CVE Search Pack v1.0.1
+
+#### Integrations
+##### __CVE Search v2__  
+Added DBot score to the ***cve*** command.
+
+---
+
+### Carbon Black Defense Pack v1.1.0
+
+#### Integrations
+##### __Carbon Black Defense__  
+ - Fixed a bug in the ***cbd-find-events*** command where hash arguments were not processed correctly.
+
+---
+
+### Carbon Black Enterprise Live Response Pack v1.0.1
+
+#### Integrations
+##### __carbonblackliveresponse__  
+Fixed an error when pulling a file in the ***get-file-from-endpoint*** command.
+
+---
+
+### Carbon Black Enterprise Response Pack v1.0.3
+
+#### Integrations
+##### __carbonblack-v2__  
+- Added new context outputs for the ***cb-get-processes*** command:
+    - **File.Name**
+    - **File.MD5**
+    - **File.Path**
+    - **Endpoint.Hostname**
+- Fixed an issue where the file context did not behave as expected in the ***cb-get-processes*** command.
+    
+
+---
+
+### Check Point Firewall Pack v1.0.1
+
+#### Integrations
+##### __Check Point__  
+Added the **command** argument to the ***checkpoint*** command. This argument is the command to run in the firewall.
+
+---
+
+### Chronicle Pack v1.0.1
+#### Integrations
+##### __Google Chronicle Backstory__  
+Integration name has been updated
+
+---
+
+
+### Claroty Pack v1.0.1
+
+#### Classifiers
+##### __Claroty - Classifier__  
+Updated classifier according to new schema.
+
+#### Integrations
+##### __Claroty__  
+Added default classifier and mapper.
+
+---
+
+### Cofense Feed Pack v1.0.2
+
+#### Integrations
+##### __Cofense Feed__  
+- Fixed a bug where the 'test' button always returned a positive 
+answer.
+- Removed the threat type **phish**.
+
+---
+
+### Cofense Triage Pack v1.1.0
+#### Integrations
+##### __Cofense Triage__  
+  - Deprecated - We recommend using Cofense Triage v2 instead (available from 5.0.0).
+##### __Cofense Triage v2__  
+    - Added support for `cofense-get-report-png-by-id` and `cofense-get-threat-indicators` commands.
+
+---
+
+
+### Common Scripts Pack v1.1.3
+#### Scripts 
+__ParseEmailFiles__ 
+- Fixed an issue parsing EML files encoded with a BOM.  
+- Fixed an issue with header parsing.
+
+##### __SetGridField__  
+- Fixed an issue with a dictionary element.
+- Added support for lists of values.
+- Added support for unpacking nested elements.
+- The argument **keys** is no longer mandatory, all the keys will be taken by default.
+##### __IsIPInRanges__  
+Added private IP address ranges.
+##### __GetTime__  
+Added support for subtracting days and years from current date using **daysAgo** and **yearsAgo** arguments.  
+
+---
+
+### Common Types Pack v1.0.1
+
+#### Classifiers
+##### __Mail Listener - Classifier__  
+Updated classifier according to new schema.
```

Comment:
lets remove all `Updated classifier according to new schema.`, its for 6.0 and above